### PR TITLE
Update more supported versions

### PR DESCRIPTION
**Pull request by tas50**
Some of these were quite out of date.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: chef_master/source/api_omnitruck.rst [hunk 1]
```diff
@@ -115,7 +115,7 @@ Omnitruck accepts the following platforms:
    * - Microsoft Windows
      - ``windows``
      - ``x86_64``
-     - ``2016``, ``2012r2``, ``2012``, ``8``
+     - ``2019``, ``2016``, ``2012r2``, ``2012``, ``8``
```
Comment (kagarmoe) [inline chef_master/source/api_omnitruck.rst]: Why do we have an ascending order in one case, but descending in the other?

**Review by kagarmoe — CHANGES_REQUESTED** [review]

File: chef_master/source/api_omnitruck.rst (hunk 1 above)
Comment (tas50) [reply chef_master/source/api_omnitruck.rst]: No clue

Comment (tas50) [reply chef_master/source/api_omnitruck.rst]: Fixed

**Review by kagarmoe — APPROVED** [review]
tenor-217295481

**Pull request merged**